src/components/base/outline-container/outline-container.css

```diff
@@ -37,62 +37,54 @@
     @apply max-w-screen-xxxl;
   }
 }
-:host([is-nested]),
-:host([is-nested='true']) {
+:host(:not([is-nested])) {
```

Comment:
This naming is confusing, but the current implementation matches the Storybook name `Padding` so that when you set `Padding` to `true`, there is padding.

If that isn't the intent, we may want to change the label in Storybook alongside this CSS rule.

Reply:
This is now updated to `x-padding`.